f8a_worker/dispatcher/nodes.yml

```diff
@@ -203,11 +174,6 @@
       selective_run_function:
         name: 'selective_run_function'
         import: 'f8a_worker.dispatcher.selective'
-    - name: 'BookkeeperTask'
```

Comment:
@pkajaba  let's not remove the BookkeeperTask because of the reason specified in the earlier comment.